enterprise_gateway/services/kernelspecs/handlers.py

```diff
@@ -52,8 +51,8 @@ async def get(self):
 
         if kernel_user:
```

Comment:
Let's move this `if` block into the one above since that's the only way `kernel_user` can have a value.

Should we check for a `None` `kernel_user` in that block?  Not sure if its possible to send something like: `GET /api/kernelspecs?user` or `GET /api/kernelspecs?user=''`, so just wondering if we should raise an error if `kernel_user` is `None` or has length == 0 when the query parameter is present?

Reply:
If we dont check if kernel_user is not empty, a call from `/api/kernelspecs?user=` will return all specs.

Is the same call `/api/kernelspecs?user=` and `/api/kernelspecs`

Reply:
We just check if kernel_user is not an empty string. If we call `/api/kernelspecs?user=''` or `/api/kernelspecs?user` , it is the same as `/api/kernelspecs`